src/frontend/src/common/api/index.tsx

```diff
@@ -99,7 +99,9 @@ const SAMPLE_MAP = new Map<string, keyof Sample>([
   ["private_identifier", "privateId"],
   ["public_identifier", "publicId"],
   ["upload_date", "uploadDate"],
+  ["uploaded_by", "uploadedBy"],
   ["sequencing_date", "sequencingDate"],
+  ["submitting_group", "submittingGroup"],
```

Comment:
To be clear, we now get this info from the API, but we still don't actually make use of it anywhere yet, right?

Reply:
correct. This definition basically says "for the keys listed in this map, covert the key from the left side (the key the BE sends) to the right side (the camel-cased equivalent, to keep things in line with JS standard). Anything not shown in the map will still be a part of the sample objects returned from the backend. But when you print the objects or look at them in dev tools, any keys not mapped here will use the snake_case version that was sent from the server.